src/gearbox/schemas/deploy.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import BaseModel
```

Comment:
This file should be moved to gearboxdatamodel

Reply:
I kept it here on purpose because it is specific to the backend. No other services will be able to stage data and migrate them.